fsharp-backend/src/LibBackend/Canvas.fs

```diff
@@ -679,15 +679,20 @@ let loadJsonFromDisk (root : Config.Root) (c : Meta) : List<tlid * PT.Oplist> =
 let loadAndResaveFromTestFile (meta : Meta) : Task<unit> =
   task {
     let oplists =
-      meta
-      |> loadJsonFromDisk Config.Testdata
-      |> List.map (fun (tlid, oplist) ->
-        let tl =
-          let oplist = fromOplist meta [] oplist
-          let tls = toplevels oplist
-          let dtls = deletedToplevels oplist
-          (Map.mergeFavoringLeft tls dtls) |> Map.get tlid |> Option.unwrapUnsafe
-        (tlid, oplist, tl, NotDeleted))
+      try
+        meta
+        |> loadJsonFromDisk Config.Testdata
+        |> List.map (fun (tlid, oplist) ->
+          let tl =
+            let oplist = fromOplist meta [] oplist
+            let tls = toplevels oplist
+            let dtls = deletedToplevels oplist
+            (Map.mergeFavoringLeft tls dtls) |> Map.get tlid |> Option.unwrapUnsafe
+          (tlid, oplist, tl, NotDeleted))
+      with
+      | ex ->
+        debuG "Couldn't find test file - assuming empty oplists" meta
```

Comment:
What's happening here?

Reply:
At one point, I believed that "tests without a test file" were failing here. It seems they're not, though, so I can remove this try/catch now.

Reply:
https://app.circleci.com/pipelines/github/darklang/dark?branch=pull%2F3485&filter=all seems I'm not crazy, actually. These failing after removing.

Reply:
Ah, it looks like the old load_json_from_disk_fluid uses `File.maybeReadFile` and uses options. Do we have a preference for Options vs Exceptions? I would imagine we prefer options.

Reply:
Agreed - just pushed a PR that gets us to a slightly nicer place